### PR TITLE
Mes 7897 unit tests

### PR DESCRIPTION
**Pull request by RLCorp**
## Description
Updated Unit Tests
## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

**Review by gittins7 — APPROVED** [review]

File: src/app/pages/journal/components/personal-commitment/__tests__/personal-commitment.spec.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+  beforeEach(waitForAsync(() => {
+    TestBed.configureTestingModule({
+      declarations: [PersonalCommitmentSlotComponent],
+      imports: [
+        IonicModule,
+      ],
+      providers: [
+        { provide: OutcomeBehaviourMapProvider, useClass: OutcomeBehaviourMapProvider },
+      ],
```
Comment (matthew2564) [inline src/app/pages/journal/components/personal-commitment/__tests__/personal-commitment.spec.ts]: Not needed.

File: src/app/pages/office/cat-adi-part2/components/vehicle-checks/__tests__/vehicle-checks-office-card.spec.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+  beforeEach(waitForAsync(() => {
+    TestBed.configureTestingModule({
+      declarations: [VehicleChecksOfficeCardCatADI2Component],
+      imports: [
+        IonicModule,
+      ],
+      providers: [
+        { provide: OutcomeBehaviourMapProvider, useClass: OutcomeBehaviourMapProvider },
+      ],
```
Comment (matthew2564) [inline src/app/pages/office/cat-adi-part2/components/vehicle-checks/__tests__/vehicle-checks-office-card.spec.ts]: Not needed.

File: src/app/pages/office/cat-cpc/components/assessment-report/__tests__/assessment-report.spec.ts [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+  beforeEach(waitForAsync(() => {
+    TestBed.configureTestingModule({
+      declarations: [AssessmentReportComponent],
+      imports: [
+        IonicModule,
+      ],
+      providers: [
+        { provide: OutcomeBehaviourMapProvider, useClass: OutcomeBehaviourMapProvider },
+      ],
```
Comment (matthew2564) [inline src/app/pages/office/cat-cpc/components/assessment-report/__tests__/assessment-report.spec.ts]: Not needed.

File: src/app/pages/office/components/independent-driving/__tests__/independent-driving.spec.ts [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+import { IndependentDrivingComponent } from '@pages/office/components/independent-driving/independent-driving';
+import {
+  ReactiveFormsModule, UntypedFormControl, UntypedFormGroup, Validators,
+} from '@angular/forms';
+import { IonicModule } from '@ionic/angular';
+import { OutcomeBehaviourMapProvider, VisibilityType } from '@providers/outcome-behaviour-map/outcome-behaviour-map';
+import { TestCategory } from '@dvsa/mes-test-schema/category-definitions/common/test-category';
+import { CircuitType } from '@shared/models/circuit-type';
+
+describe('DangerousFaultBadgeComponent', () => {
```
Comment (matthew2564) [inline src/app/pages/office/components/independent-driving/__tests__/independent-driving.spec.ts]: Describe should be for 'IndependentDrivingComponent'.

Same issue here ```src/components/common/adi3-debrief-card/__tests__/adi3-debrief-card.spec.ts```

File: src/app/pages/office/components/true-likeness/__tests__/true-likeness.spec.ts [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+    TestBed.configureTestingModule({
+      declarations: [TrueLikenessComponent],
+      imports: [
+        IonicModule,
+        ReactiveFormsModule,
+      ],
+      providers: [
+        provideMockStore({ ...{} }),
+        { provide: OutcomeBehaviourMapProvider, useClass: OutcomeBehaviourMapProvider },
+      ],
```
Comment (matthew2564) [inline src/app/pages/office/components/true-likeness/__tests__/true-likeness.spec.ts]: Not needed.

File: src/app/pages/office/components/vehicle-checks/__tests__/vehicle-checks-office-card.spec.ts [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+  beforeEach(waitForAsync(() => {
+    TestBed.configureTestingModule({
+      declarations: [VehicleChecksOfficeCardComponent],
+      imports: [
+        IonicModule,
+      ],
+      providers: [
+        { provide: OutcomeBehaviourMapProvider, useClass: OutcomeBehaviourMapProvider },
+      ],
```
Comment (matthew2564) [inline src/app/pages/office/components/vehicle-checks/__tests__/vehicle-checks-office-card.spec.ts]: Not needed.

File: src/app/pages/test-report-dashboard/components/review-feedback/__tests__/review-feedback.spec.ts [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+  beforeEach(waitForAsync(() => {
+    TestBed.configureTestingModule({
+      declarations: [ReviewFeedback],
+      imports: [
+        IonicModule,
+      ],
+      providers: [
+        provideMockStore({ ...{} }),
+
+      ],
```
Comment (matthew2564) [inline src/app/pages/test-report-dashboard/components/review-feedback/__tests__/review-feedback.spec.ts]: Not needed.

File: src/app/pages/test-report/cat-adi-part3/components/lesson-theme/__tests__/lesson-theme.spec.ts [hunk 8]
```diff
@@ -0,0 +1,129 @@
...
+  beforeEach(waitForAsync(() => {
+    TestBed.configureTestingModule({
+      declarations: [LessonThemeComponent],
+      imports: [
+        IonicModule,
+      ],
+      providers: [
+        provideMockStore({ ...{} }),
+
+      ],
```
Comment (matthew2564) [inline src/app/pages/test-report/cat-adi-part3/components/lesson-theme/__tests__/lesson-theme.spec.ts]: Not needed.

File: src/components/common/data-row-custom/__tests__/data-row-custom.spec.ts [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+
+  beforeEach(waitForAsync(() => {
+    TestBed.configureTestingModule({
+      imports: [
+        IonicModule,
+      ],
+      providers: [
+        { provide: OutcomeBehaviourMapProvider, useClass: OutcomeBehaviourMapProvider },
+        { provide: NavParams, useClass: NavParamsMock },
+      ],
```
Comment (matthew2564) [inline src/components/common/data-row-custom/__tests__/data-row-custom.spec.ts]: Not needed.

File: src/app/pages/journal/components/personal-commitment/__tests__/personal-commitment.spec.ts (hunk 1 above)
Comment (RLCorp) [reply src/app/pages/journal/components/personal-commitment/__tests__/personal-commitment.spec.ts]: fixed in newest commit

File: src/app/pages/office/cat-adi-part2/components/vehicle-checks/__tests__/vehicle-checks-office-card.spec.ts (hunk 2 above)
Comment (RLCorp) [reply src/app/pages/office/cat-adi-part2/components/vehicle-checks/__tests__/vehicle-checks-office-card.spec.ts]: fixed in newest commit

File: src/app/pages/office/cat-cpc/components/assessment-report/__tests__/assessment-report.spec.ts (hunk 3 above)
Comment (RLCorp) [reply src/app/pages/office/cat-cpc/components/assessment-report/__tests__/assessment-report.spec.ts]: fixed in newest commit

File: src/app/pages/office/components/independent-driving/__tests__/independent-driving.spec.ts (hunk 4 above)
Comment (RLCorp) [reply src/app/pages/office/components/independent-driving/__tests__/independent-driving.spec.ts]: fixed in newest commit

File: src/app/pages/office/components/true-likeness/__tests__/true-likeness.spec.ts (hunk 5 above)
Comment (RLCorp) [reply src/app/pages/office/components/true-likeness/__tests__/true-likeness.spec.ts]: fixed in newest commit

File: src/app/pages/office/components/vehicle-checks/__tests__/vehicle-checks-office-card.spec.ts (hunk 6 above)
Comment (RLCorp) [reply src/app/pages/office/components/vehicle-checks/__tests__/vehicle-checks-office-card.spec.ts]: fixed in newest commit

File: src/app/pages/test-report-dashboard/components/review-feedback/__tests__/review-feedback.spec.ts (hunk 7 above)
Comment (RLCorp) [reply src/app/pages/test-report-dashboard/components/review-feedback/__tests__/review-feedback.spec.ts]: fixed in newest commit

File: src/app/pages/test-report/cat-adi-part3/components/lesson-theme/__tests__/lesson-theme.spec.ts (hunk 8 above)
Comment (RLCorp) [reply src/app/pages/test-report/cat-adi-part3/components/lesson-theme/__tests__/lesson-theme.spec.ts]: fixed in newest commit

File: src/components/common/data-row-custom/__tests__/data-row-custom.spec.ts (hunk 9 above)
Comment (RLCorp) [reply src/components/common/data-row-custom/__tests__/data-row-custom.spec.ts]: fixed in newest commit

**Review by matthew2564 — APPROVED** [review]

**Pull request merged**